pandas/core/series.py

```diff
@@ -5087,7 +5088,8 @@ def _arith_method(self, other, op):
 
         lvalues = self._values
         rvalues = extract_array(other, extract_numpy=True)
-        result = ops.arithmetic_op(lvalues, rvalues, op)
+        with np.errstate(all="ignore"):
+            result = ops.arithmetic_op(lvalues, rvalues, op)
```

Comment:
do we not need this for _logical_method?

Reply:
Currently we don't suppress any warnings in `ops.logical_op`, so I left the current behaviour as is.

Reply:
makes sense